lib/src/widgets/settings_widgets.dart

```diff
@@ -1260,8 +1260,8 @@ class DropDownSettingsTile<T> extends StatefulWidget {
   /// Selected value in the radio button group otherwise known as group value
   final T selected;
 
-  /// A map containing unique values along with the display name
-  final Map<T, String> values;
+  /// A map containing unique values which could use a String of a Widget as the display value
+  final Map<T, dynamic> values;
```

Comment:
@enchance  As mentioned in the following comment, we should revert this back to Map<T, String> values, and then use the string values to decide what to render inside the optional `itemBuilder` function.

Since, the `itemBuilder` is optional, we can without an issue render the displayNames as Text